### DISCUSSION TITLE
DCP data in DIGGS?

### DISCUSSION THREAD

**Discussion by staliaferro** [DIGGSml/schema-dev#23] [Q&A] [answered]
Ohio DOT has started using Automatic DCP in conjunction with SPT on large roadway projects as well as using only DCP on small intersection projects.  Are there provisions in DIGGS for DCP?

**Comment by staliaferro** [comment, marked as answer]
Suggestion from Peter Narsavage was to use DrivenPenetrationTest element.

**Reply by panars** [reply]
The DrivenPenetrationTest element is general enough that I think it could be used for the DCP. It is intended for the standard penetration test, Becker hammer test, and other tests where a sampler is driven by impacts from a hammer. But I don't think retrieving a sample is required.  We should have a discussion whether it is better to use an existing DIGGs element for the DCP or create a new element for it. In my opinion, I think the default should be to use existing elements at the start, unless (or until) the limitations of using the existing element justify the creation of a new element.   This also leads me to another question about standardizing the values for penetrationTestType, which I will post in another discussion.

**Reply by sdeaton75** [reply]
I agree - from reviewing the documentation this element is general enough to handle SPT and Dynamic Probe (in the UK).  So, I believe its intent was to be used to support these various methods because the Dynamic Probe group in the AGS has the items such as torque and delay which apply to this test but not the SPT.

**Reply by dponti** [reply]
This is correct. The intent to try to limit schema bloat by defining test procedure objects that are generalized enough to handle variants of a type of test procedure without requiring new schema objects that potentially will duplicate many of the same properties. Additionally, more specific objects or extensions of objects can be defined if there are limitations in the current objects that make utilization of them impractical or inappropriate.
